modules/led.sh

```diff
@@ -38,6 +38,12 @@ function led {
     echo "Look at your RPi leds, both leds will be in this pattern... "
     echo "Both LED: 1 sec on; 8 blink; 1 on"
     christmas > /dev/null
+  elif [ "$color" = "carneval" ]; then
+    checkroot
+    echo "leds are set to carnval mode."
+    echo "Look at your RPi leds, both leds will be in this pattern... "
+    echo "Both LED: 1 sec on; 8 blink; 1 on"
```

Comment:
You should change the timing descriptions to match yours.

Reply:
Also, `carneval` is spelled `carnival` unless you're referring to something else.

Reply:
Hi kjong, I corrected the spell ( carnival ) . thanks for letting me know.

Reply:
I also change the timing descriptions